Doc/library/asyncio-task.rst

```diff
@@ -537,6 +555,93 @@ Shielding From Cancellation
 Timeouts
 ========
 
+.. coroutinefunction:: timeout(delay)
+
+    A convenient way to limit the amount of time spent waiting on
+    something is to use the :func:`asyncio.timeout`
+    :ref:`asynchronous context manager <async-context-managers>`.
```

Comment:
Now that this paragraph is part of the function description it shouldn't refer to itself. Maybe "An _asynchronous context manager_ that can be used to limit ..."?